cookbooks/aws-parallelcluster-environment/test/controls/system_authentication_spec.rb

```diff
@@ -45,9 +45,11 @@
   end
 end
 
-control 'system_authentication_configured' do
+control 'tag:config_system_authentication_configured' do
   title 'Check that system authentication is configured correctly'
 
+  only_if { !os_properties.on_docker? }
```

Comment:
Why are we adding this condition for testing only on EC2?

Reply:
Because the code isn't able to work correctly on docker. For instance, services don't work correctly on it.